src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/hearing/HearingNoticeHmcGenerator.java

```diff
@@ -86,7 +87,7 @@ public HearingNoticeHmc getHearingNoticeTemplateData(CaseData caseData, HearingG
 
         return HearingNoticeHmc.builder()
             .title(getHearingTypeTitleText(caseData, hearing))
```

Comment:
I'm pretty sure this getHearingTypeTitleText method calls the below method with hardcoded 'hearing' as a value:
![image](https://github.com/user-attachments/assets/9590cde3-3334-42ad-889c-dc740a3a0b45)
You might have to add a Welsh version that it can use instead for the Welsh notice.